src/components/SelectInput/SelectInput.test.jsx

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import Select from 'react-select';
+import '@babel/polyfill';
+import 'mutationobserver-shim';
```

Comment:
Both of these imports should be happening in setupTests.js, but putting them there had no effect. Could someone guess why?

Reply:
@JCarmine I believe these should both be in the .babelrc rather than the setupTests file

Reply:
@JCarmine by bad I think you are correct with your initial assessment but I am not sure yet why the `setupTests` file is not loading these dependencies properly. Will continue to investigate.